about.html

```diff
@@ -0,0 +1,165 @@
+          </span>
+        </p>
+      </div>
+      <p class="contact-us-text">Please contact us with email for any further questions about World Tomato Forum 2023!
+      </p>
+      <a id="contact-email" href="#">wtfforum2023@wtf.com</a>
+    </div>
+  </section>
+  <!-- logo section -->
+  <section class="our-logo">
+    <div class="container">
+      <header class="our-logo-header">
+        <h2 id="logo-title">The World Tomato Forum 2023 Logo</h2>
+        <hr class="our-logo-hr">
+      </header>
+      <div class="our-logo-body">
+        <p id="our-logo-text">The logo of The World Tomato Forum 2023 was not decided through the logo competition in 2022.<span
+            id="logo-text-desktop">
+            The logo was designed by a group of students from the University of Sevilla. They were awarded with 1 ton of
+            organic tomatoes.
+          </span>
+        </p>
+        <img id="our-logo-image" src="assets/world_tomato_forum_logo.svg" alt="">
+      </div>
+    </div>
+  </section>
+  <!-- logo section -->
+
+
+  <section class="previous-events">
+    <div class="container">
+      <header class="previous-events-header">
+        <h2>See the past WT Forum Events</h2>
+        <hr class="our-logo-hr">
+      </header>
+      <p id="previous-events-supporting">Take a look at the our past two events!</p>
+      <div class="previous-events-body">
+
+        <a class="previous-event-img-container" href="#">
+          <p>
+            <strong>2022</strong> <br> World Tomato Forum
+          </p>
+        </a>
+
+        <a class="previous-event-img-container prev-second-img" href="#">
+          <p>
+            <strong>2021</strong> <br> World Tomato Forum
+          </p>
+        </a>
+
+      </div>
+    </div>
+
+  </section>
+
+
+  <section class="partners-section">
+    <div class="container">
+
+      <header class="partners-section-header">
+        <h2 id="partner-header-title">Partners</h2>
+        <hr class="partners-section-header-hr">
+      </header>
+      <div class="partners-body-container">
+        <img class="partners-logo" src="assets/partners_logo/partners_bill_melinda_gates.png" alt="">
+        <img class="partners-logo" src="assets/partners_logo/partners_eu_bw.png" alt="">
+        <img class="partners-logo" src="assets/partners_logo/partners_generalitat_valenciana_bw.png" alt="">
+        <img class="partners-logo" src="assets/partners_logo/partners_spanish_consulate_bw.png" alt="">
+        <img class="partners-logo" src="assets/partners_logo/partners_wef_bw.png" alt="">
+      </div>
+    </div>
+  </section>
+
+
+
+  <footer>
+    <div class="container footer-container">
+      <img id="footer-logo" src="assets/world_tomato_forum_logo.svg" alt="">
+      <p class="small-footer-text">2023 World Tomato Forum. All Rights Reserved. <br><span class="footer-text-desktop">
+        세계 토마토 포럼은 세계 포럼의 기초입니다. 저작권 문제는 이메일 주소로 문의하시기 바랍니다.
+        </span></p>
+
+    </div>
+  </footer>
```

Comment:
- [ ] Great job so far, In the design the about page, desktop version, has different styling and layout than the home page, please fix that and also the about page doesn't contain the partner section.
- [ ] the background color should be **#272a31**, kindly fix that.

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/163164777-70530281-c091-45b0-a18b-f6b48e784b70.png)

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/163164883-8c87714b-7de9-4718-a20b-a2918bfabfbc.png)

Reply:
Great job so far, In the design the about page, desktop version, has different styling and layout than the home page, please fix that and also the about page doesn't contain the partner section.
-Kindly check the design. It has a partners section on the mobile version on the about page.